src/final2/subtests/MoveTest.java

```diff
@@ -12,16 +16,20 @@
  */
 public class MoveTest extends LangtonSubtest {
 
+	public MoveTest() {
+		setAllowedSystemExitStatus(SystemExitStatus.WITH_0);
+	}
+
 	/**
 	 * Asserts that {@code move} works on a very simple, easy to understand example: The rule is set to
 	 * {@code 90-90-90-90-90} and we have only one ant on this 2x2 board:
 	 * 
-	 * <pre>
 	 * <code>
+	 * <pre>
 	 * 0a
 	 * 00
-	 * </code>
 	 * </pre>
+	 * </code>
```

Comment:
I checked again, obviously I'm in the wrong here and `<pre>` does enable the monospace font. (Don't know why I was so sure that it didn't).

> If anything, it should be `<pre><code>`

No, it should not. Because I can't convince the formatter not to break after `<code>`. Therefore, this order inserts one newline before and after the block that was not intended. That's why I switched it.

But as `<pre>` works as expected, I'll switch to only using `<pre>`.